prime-router/src/main/kotlin/serializers/RedoxSerializer.kt

```diff
@@ -0,0 +1,181 @@
+package gov.cdc.prime.router.serializers
```

Comment:
I did a quick sanity check - Generate some redox json, and did a quick eyeball sanity check in a prettyprinter.   All looks good so far.